modules/orchestrator/proc-install-components-for-orchestrator-plugin.adoc

```diff
@@ -71,4 +66,9 @@ $ ./plugin-infra.sh
 ----
 
 .{product-very-short} Orchestrator Infra Helm Chart
-You can use {product-very-short} Orchestrator Infra Helm Chart to install components for the Orchestrator plugins in empty clusters.
\ No newline at end of file
+You can use {product-very-short} Orchestrator Infra Helm Chart to install components for the Orchestrator plugins in empty clusters.
+
+[IMPORTANT]
+====
+The Orchestrator Infra Helm chart is suitable for production use. By default, it does *not* auto-install or auto-upgrade the Operator: you must approve the install plans.
```

Comment:
```suggestion
The Orchestrator Infra Helm chart is suitable for production use. By default, it does *not* auto-install or auto-upgrade the required Serverless Operators: you or a cluster administrator must approve the install plans.
```

Just to clarify that this is only about the Serverless Operators, not the RHDH Operator.